src/services/laodeai/index.js

```diff
@@ -81,19 +81,19 @@ async function laodeai(context) {
   }
 
   const $ = cheerio.load(ddgBody);
-  const validSources = $('.web-result')
-    .map((_, el) => {
+  const sources = $('.web-result').get();
+  if (sources.length <= 1 && $(sources[0]).find('.no-results').get()) {
+    await context.reply("Uhh, I don't have an answer for that, sorry.");
+    return;
+  }
+
+  const validSources = sources
+    .map((el) => {
```

Comment:
Shouldn't we still need to handle if they're still no valid sources available before line 97?

Reply:
sek, maksudnya gimana, aku gagal paham

Reply:
itu kan di line atasnya kamu cuma handle kalo DDG emang ga punya result.

tapi dibawahnya kan nyari validSources sebelom di scrape lagi links nya sama DDG. what if there are no available valid sources?

Reply:
oh iya lupa, ok